pkg/controllers/sync/dispatch/managed.go

```diff
@@ -55,24 +51,16 @@ const IndexRolloutPlans = "federation_placement_rollout"
 // interface required for dispatching operations to managed resources.
 type FederatedResourceForDispatch interface {
 	TargetName() common.QualifiedName
-	TargetKind() string
 	TargetGVK() schema.GroupVersionKind
+	TargetGVR() schema.GroupVersionResource
```

Comment:
Are these methods necessary? We can derive them quite easily from either the FTC or the `GenericFederatedObject`

Reply:
Not strictly, but I don't see a point in removing them either.